src/v2/view-builder/views/ov/NumberChallengePushView.js

```diff
@@ -0,0 +1,122 @@
+import { _, loc, View } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+import BaseForm from '../../internals/BaseForm';
+import polling from '../shared/polling';
+import { WARNING_TIMEOUT } from '../../utils/Constants';
+
+let correctAnswer = '';
+const warningTemplate = View.extend({
```

Comment:
I would rather create their own views and handle show/hide/delay in those views like `ResendView.js` ?